helm/chaos-mesh/templates/_certs.tpl

```diff
@@ -0,0 +1,141 @@
+
+{{/*
+webhook.apiversion is used to take care compatibility with admissionregistration.k8s.io api groups
+
+When using this template, it requires the top-level scope
+*/}}
+{{- define "webhook.apiVersion" -}}
+  {{- $webhookApiVersion := "v1beta1" -}}
+  {{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" -}}
+    {{- $webhookApiVersion = "v1" -}}
+  {{- end -}}
+  {{- printf "admissionregistration.k8s.io/%s" $webhookApiVersion -}}
+{{- end -}}
+
+{{/*
+chaosmesh.selfSignedCABundleCertPEM is the self-signed CA to:
+- sign the certification keyparing used by chaos-daemon mTLS
+- sign the certification keyparing used by webhook server (if the user does not provide one)
+*/}}
+{{- define "chaosmesh.selfSignedCABundleCertPEM" -}}
+  {{- $caKeypair := .selfSignedCAKeypair | default (genCA "chaos-mesh-ca" 1825) -}}
+  {{- $_ := set . "selfSignedCAKeypair" $caKeypair -}}
```

Comment:
I think the `caKeypair` in `chaosmesh.selfSignedCABundleCertPEM` and `webhook.caBundleCertPEM` could use different key to save, like `chaosmeshSelfSignedCAKeypair` and `webhookSelfSignedCAKeypair`. 🤔

Reply:
Do you mean we prefer to use (at most) 3 different self-signed CA for

- webhooks for CRD
- chaos-daemon mtls
- chaosd mtls

I am OK with that. 

But I do not prefer to do that in this PR, this PR is a "refactor" one, try not to change the original logic. ❤️

I would update this into #3015.

Reply:
> Do you mean we prefer to use (at most) 3 different self-signed CA for
> 
> * webhooks for CRD
> * chaos-daemon mtls
> * chaosd mtls

Yes!